src/common/config.cc

```diff
@@ -949,6 +953,10 @@ int md_config_t::_get_val(const char *key, char **buf, int len) const
     ostringstream oss;
     if (bool *flagp = boost::get<bool>(&cval)) {
       oss << (*flagp ? "true" : "false");
+    } else if (float *fp = boost::get<float>(&cval)) {
```

Comment:
can we refactor this method to reuse `md_config_t::_get_val(const char *key, std::string *value)`?